helm/oncall/templates/_env.tpl

```diff
@@ -484,6 +504,22 @@
 {{- end }}
 {{- end }}
 
+{{- define "snippet.redis.port" -}}
+{{ if not .Values.redis.enabled -}}
+  {{ default 6379 .Values.externalRedis.port | quote }}
+{{- else -}}
+  "6379"
+{{- end }}
+{{- end }}
+
+{{- define "snippet.redis.database" -}}
+{{ if not .Values.redis.enabled -}}
+  {{ default 0 .Values.externalRedis.database | quote }}
+{{- else -}}
+  "0"
+{{- end }}
+{{- end }}
+
```

Comment:
for these three new `define` blocks, I would remove the leading spaces before the values (ex. ` "0"` vs `"0"`

Reply:
I just follow the existing code style here. The indent will not be placed to manifest after templating. In `"snippet.rabbitmq.password.secret.key"` default value is set with indent. Should I replace it everywhere in the chart?